### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_ Spark Local Console-Locally run_Screen Reader: Jaws is not reading the selected data present in the dropdown “Use classpath of module”.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3617]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS1.3.1; #ScreenReader; #Jaws; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28

Screen Reader: JAWS.
MAS Violated: MAS1.3.1

**Repro Steps:**
1.Turn on Screen Reader.
2.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
3.Launch IntelliJ IDEA.
4.Tab till Create New Project button and press enter
5.Tab till Azure Spark/HDInsight-->Spark Project Scala
6.Tab till Build Tools-->Maven
7.Tab till Next button and press enter
8.Tab till finish button and press enter
9. Navigate to Run --> Edit Configurations-->Apache Spark on HDInsight > [Spark on HDInsight] myApp
10.Tab till Locally run tab.
11.Tab till “Use classpath of module” dropdown.
12.Verify whether Jaws is reading the selected names present in the dropdown “Use classpath of module” or not.

**Actual Result:**
Jaws is not reading the selected data and also items present in the dropdown “Use classpath of module”.

**Expected Result:**
Jaws should read the selected data present in the dropdown “Use classpath of module”, When focus is pointing on it.

**User Impact:**
Screen Reader users will face difficulty in knowing what data is selected if jaws is not reading any information that was present in the dropdown.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_Jaws is not reading selected data for class path.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646878/Bug_Jaws.is.not.reading.selected.data.for.class.path.pptx)

**Comment by t-rufang** [comment]
Note: screen reader is not reading the selected item on the combobox `Use class path of module`.
![image](https://user-images.githubusercontent.com/32627233/65662656-627b1e80-e067-11e9-9dd9-59e112642480.png)

**Comment by t-rufang** [comment]
The `Use classpath of module` code is reused from IntelliJ code and it's out of the scope of our toolkit. Please find the same bug in the `application` run configuration page. 
![image](https://user-images.githubusercontent.com/32627233/67267613-3d05f700-f4e5-11e9-9ace-f4fa972b8699.png)
When you tab to the `Use classpath of module` combobox. The JAWS is not reading the `untitle8` in the combobox.

**Comment by t-rufang** [comment]
- Importance of experience: Minimal. JAWS is not reading the content in `Use ckasspath of module` combo box. Most of the time user will use the default module of a project and they don't need to change this field to switch to other module.  
- Quality of experience: Blocking. Keyboard user can use up and down arrow to switch to another module but JAWS is not reading the content of each item.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by v-lasiva** [comment]
As per discussion with SME, Closing the issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749717
